nvflare/client/ex_process/api.py

```diff
@@ -81,20 +82,23 @@ def init(self, rank: Optional[str] = None):
             rank = os.environ.get("RANK", "0")
 
         if self.process_model_registry:
-            print("Warning: called init() more than once. The subsequence calls are ignored")
+            logging.warn("Warning: called init() more than once. The subsequence calls are ignored")
```

Comment:
Change to use self.logger.warning(). 

from nvflare.fuel.utils.obj_utils import get_logger
self.logger = get_logger(self)

Reply:
thanks, changed as suggested